doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -548,6 +548,7 @@ This section provides detailed lists of changes by :ref:`script <scripts>`.
 * :ref:`west_sbom`:
 
   * Output now contains source repository and version information for each file.
+  * The script now supports Matter because it can enter GN external projects and get the source files from it.
```

Comment:
```suggestion
  * The script now supports Matter because it has access to GN external projects and can get the source files from them.
```
them = GN external projects, did I get this right?